src/components/Dashboard.tsx

```diff
@@ -11,10 +12,10 @@ export const Dashboard = () => {
     collection: `users/${uid}/columns`,
     storeAs: 'columns',
   })
-  const columns = useSelector(
+  const columndData = useSelector(
```

Comment:
`columndData`? 🤔 😸

Reply:
intentionally typo to check if you are reading this